survey/survey_app/cron.py

```diff
@@ -4,7 +4,7 @@
 from django_cron import CronJobBase, Schedule
 
 class DailySurveySubmissionsEmail(CronJobBase):
```

Comment:
https://django-extensions.readthedocs.io/en/latest/jobs_scheduling.html
We often use this for job scheduling @newDaria

Reply:
so should I move  cron.py to?
survey_app -> jobs -> daily -> cron.py